src/languages/_french.json

```diff
@@ -1752,7 +1752,10 @@
         "transaction_canceled": "Transaction canceled! :)",
         "transaction_sent": "Transaction sent! :)"
       },
-      "unknown_application": "Unknown Application :)"
+      "unknown_application": "Unknown Application :)",
+      "connection_failed": "Connection failed",
```

Comment:
@skylarbarrera to confirm; iirc we'd added the `:)` token to the end of strings in the _french.json that are still in english

Reply:
Ah I did not know that. There are a bunch in here at this point without smileys I think.